packages/connection/tests/php/test-rest-endpoints.php

```diff
@@ -167,10 +226,12 @@ public function test_connection_reconnect_full() {
 
 		$data = $response->get_data();
 		$this->assertEquals( 'in_progress', $data['status'] );
-		$this->assertSame( 0, strpos( $data['authorizeUrl'], 'https://jetpack.wordpress.com/jetpack.authorize/1' ) );
+		$this->assertSame( 0, strpos( $data['authorizeUrl'], 'https://jetpack.wordpress.com/jetpack.authorize/' ) );
```

Comment:
What do you think of moving all of the assert statements to the end of the test method, so that the cleanup stuff (removing filters, calling `shutdown_reconnect_test`) will run even if an assert fails?